requests_pkcs12.py

```diff
@@ -169,6 +170,30 @@ def post(*args, **kwargs):
 def put(*args, **kwargs):
     return request('put', *args, **kwargs)
 
+def execute_test_case(test_case, key, cert):
+    password = test_case['pkcs12_password']
+    try:
+        algorithm = cryptography.hazmat.primitives.serialization.BestAvailableEncryption(password) \
+            if test_case['pkcs12_password'] is not None else cryptography.hazmat.primitives.serialization.NoEncryption()
+        pkcs12_data = cryptography.hazmat.primitives.serialization.pkcs12.serialize_key_and_certificates(
+            name=b'test',
+            key=key,
+            cert=cert,
+            cas=[cert, cert, cert],
+            encryption_algorithm=algorithm
+        )
+        response = get(
+            'https://example.com/',
+            pkcs12_data=pkcs12_data,
+            pkcs12_password=test_case['pkcs12_password']
+        )
+        if response.status_code != test_case['expected_status_code']:
+            raise Exception('Unexpected response: {response!r}'.format(**locals()))
+    except ValueError as e:
+        if test_case['expected_exception_message'] is None or str(e) != test_case['expected_exception_message']:
+            raise(e)
+
```

Comment:
Coding style: Please fix the double empty line to a single empty line.

Reply:
Sorry, don't get it. Please fix rest of minor changes as you wish :)